docs/graalvm-m1-mac.md

```diff
@@ -0,0 +1,45 @@
+## Installing GraalVM on M1 Mac
```

Comment:
Is this specific to M1 macs only?

Reply:
Yes, you can install using `sdkman` (or manually) on other platforms without any additional actions.